sktime/forecasting/base/_adapters.py

```diff
@@ -12,6 +12,72 @@
 from sktime.forecasting.base._sktime import _SktimeForecaster
 from sktime.forecasting.base._sktime import _OptionalForecastingHorizonMixin
 
+from sktime.utils.validation.forecasting import check_y
+
+
+class _TbatsAdapter(_OptionalForecastingHorizonMixin, _SktimeForecaster):
+    """Base class for interfacing tbats forecasting algorithms"""
+
+    def fit(self, y, X=None, fh=None):
+        """Fit to training data.
+
+        Parameters
+        ----------
+        y : pd.Series
+            Target time series to which to fit the forecaster.
+        fh : int, list or np.array, optional (default=None)
+            The forecasters horizon with the steps ahead to to predict.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables (ignored)
+
+        Returns
+        -------
+        self : returns an instance of self.
+        """
+        if X is not None:
+            raise NotImplementedError("BATS/TBATS don't support exog or endog data.")
+
+        y = check_y(y)
+        self._set_y_X(y, X)
+        self._set_fh(fh)
+
+        self._forecaster = self._forecaster.fit(y)
+        self._is_fitted = True
+        return self
+
+    def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+        if not fh.is_relative:
+            fh = fh.to_relative(cutoff=self.cutoff)
```

Comment:
No need to check for if it is relative, that's handled internally. So you can simply call `fh = fh.to_relative(cutoff=self.cutoff)` and if it is relative already it ignores the cutoff and just returns `self`.